src/eme.js

```diff
@@ -178,10 +178,13 @@ const defaultGetLicense = (url) => (emeOptions, keyMessage, callback) => {
   });
 };
 
-const promisifyGetLicense = (getLicenseFn) => {
+const promisifyGetLicense = (getLicenseFn, player) => {
   return (emeOptions, keyMessage) => {
     return new Promise((resolve, reject) => {
       getLicenseFn(emeOptions, keyMessage, (err, license) => {
+        if (player && player.tech_) {
+          player.trigger('licenseRequestAttempted');
```

Comment:
I think we may want to make this all lowercase to match other events

Reply:
Ah, I can change this, I thought they were camel case, but I probably misread that.